_pages/en_US/troubleshooting.txt

```diff
@@ -155,3 +155,16 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Your device is bricked
 1. You will need to get an ntrboot-comptible flashcart (one of the ones on [this list](ntrboot) or a [hardmod](https://gbatemp.net/threads/414498/), or repair / replace your device
 1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
+
+## System update error after installing CFW
+
+  + You get an error while trying to use WiFi (during system update)
+  + reboot your 3DS after each option and try updating again
+1. set your DNS settings to "Auto"
+1. move closer to your WiFi router
+1. delete your WiFi connection, then connect to your WiFi again
```

Comment:
A bit awkwardly worded IMO. 'm partial to using the term `reconnect` but it might be unnecessarily vague.